mne/channels/tests/test_montage.py

```diff
@@ -281,6 +282,12 @@ def test_montage():
     assert_allclose(oz[0], 0, atol=1e-2)  # midline
     assert_true(f1[0] < 0 and f1[1] > 0)  # left frontal
 
+    # test get_builtin_montages function
+    montages = get_builtin_montages()
+    assert_true(len(montages) > 0)  # MNE should always ship with montages
+    assert_true("standard_1020" in montages)  # 10/20 montage
+    assert_true("standard_1005" in montages)  # 10/05 montage
```

Comment:
At some point it would be a fun exercise to add a test that `doc/manual/whatever.rst` has the same list of montages that are in this list. If you're feeling bored feel free to do it in a subsequent PR.